glean-core/ios/Glean/Net/HttpPingUploader.swift

```diff
@@ -21,10 +26,8 @@ public class HttpPingUploader {
     ///     * path: The URL path to append to the server address
     ///     * data: The serialized text data to send
     ///     * config: The Glean `Configuration` object
-    func logPing(path: String, data: String, config: Configuration) {
-        if config.logPings {
-            NSLog("\(Constants.logTag) : \(path)\n\(data)")
-        }
+    func logPing(path: String, data: String) {
+        NSLog("\(Constants.logTag) : \(path)\n\(data)")
```

Comment:
We're printing this message slightly differently compared to the Android implementation: we are printing a JSON indented version of the string, not the raw string there. Can you file a bug for looking up how to achieve the same result (or if it even make sense to do so, given how logs are consumed on iOS?)?

Reply:
I'm going to drop a comment in [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1580856) regarding this.  I think that the result of that bug should give us a logging class that we can use on iOS to achieve closer results to Android loggging.